### ISSUE TITLE
Cop Idea: Replace #send(LITERAL, ...) with method call

### ISSUE THREAD

**Issue by jdufresne** [rubocop/rubocop#12842]
## Is your feature request related to a problem? Please describe.

In projects I've worked on, I've seen cases where `#send` was called with a literal symbol or string. This can come about due to heavy refactoring where the send was once more dynamic.

For example:

```rb
# bad
send(:puts)
send('puts')
obj.send(:nil?)
obj.send('nil?')

# good
puts
obj.nil?
```

## Describe the solution you'd like

A new cop that detects `#send(LITERAL, ...)` and replaces it with the method call named by the literal.

## Describe alternatives you've considered

The alternative is not implementing the cop and closing this issue. As I discover these by inspection, I manually fix them.

## Additional context

N/A

**Comment by Earlopain** [comment]
`send` ignores method visibility. This would only really work for `public_send` I think.

**Comment by jdufresne** [comment]
That is a good point. I suppose the cop could be configurable to apply dangerously to all `send` or safely to only `public_send`. Be default, it could be safe.
